cryostatio/cryostat-web
# build(deps): revert @patternfly/react-table from 4.112.39 to 4.112.6

This reverts commit 943460c53b4c5865c34abfa65c3b8528c03e0dd5.

Signed-off-by: Andrew Azores <aazores@redhat.com>

# Welcome to Cryostat! 👋
## Before contributing, make sure you have:
_______________________________________________

Related to #863

## Description of the change:
This reverts #858 , which introduced some styling change that causes views containing tables to render strangely on 1920x1200 displays (or anything not 16:9 perhaps).

## Motivation for the change:
This is a short-term fix for the bug referenced. Ideally we can upgrade to that `@patternfly/react-table` version or a later one. There may be some css fix or similar that we need to apply locally on top of the version upgrade.

## How to manually test:
Simply resizing the viewport doesn't seem to trigger the bug, so I'm not sure how to test this without having access to a non-16:9 display. With such a display, just open the web-client and go to ex. `Recordings` and check if the negative space at the bottom of the page is present.
